tests/unit/Directors.spec.ts

```diff
@@ -34,258 +23,456 @@ async function click (vm: any, id: string) {
   await Vue.nextTick()
 }
 
-function setValue (vm: any, id: string, value: string) {
+async function setValue (vm: any, id: string, value: string) {
```

Comment:
Yeah, sorry, this file was pretty much a rewrite -- but you know I looked at it closely! 😉